src/utils/constants/packageAccounts.ts

```diff
@@ -0,0 +1,44 @@
+import { env } from "../../config";
+
+type AccountType = Record< "name" | "description" | "fee", string | string |string>;
+
+type AccountTypes = Record<string, AccountType>;
+
+const PackageAccounts: AccountTypes =  {
```

Comment:
I think this should call this something like PackageTypeDetails, make the name more descriptive